src/sql/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -345,6 +339,11 @@ configurationRegistry.registerConfiguration({
 			'type': 'boolean',
 			'default': false,
 			'description': localize('notebook.enableIncrementalGridRendering', "Enable incremental grid rendering for notebooks. This will improve the initial rendering time for large notebooks. There may be performance issues when interacting with the notebook while the rest of the grids are rendering.")
+		},
+		useNewMarkdownRendererKey: {
+			'type': 'boolean',
+			default: true,
+			'description': localize('notebook.useNewMarkdownRenderer', "Whether to use the newer version of the markdown renderer for Notebooks.")
```

Comment:
how about instead of "newer" we say something like "match the VS Code markdown renderer" or something to that effect?

Reply:
I think it'd be better to not tie ourselves to that. For one we may make our own changes (especially given the current set of issues). And also I'm not really sure people would understand what that would mean - since we usually wouldn't have the latest it would be easy for people to be confused if something behaved differently.

But we can discuss options - is there any specific reason you want to add something like that?